src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -200,7 +200,9 @@ public function configureOptions(OptionsResolver $resolver)
      * return the value related to FieldDescription, if the associated object does no
      * exists => a temporary one is created.
      *
-     * @param object $object
+     * NEXT_MAJOR: Change signature for (object $object, FieldDescriptionInterface $fieldDescription).
```

Comment:
Not sure about this new signature, passing null before gets null as a response, but with this new signature on master will give error (not saying you should change it, we need to discuss it)

Reply:
ok, waiting for other contributors than

Reply:
can't find anything at the moment.

But I'm more PHP_CodeSniffer guy

Reply:
@VincentLanglet @OskarStark @greg0ire @core23 
what do you guys think about this?

Reply:
For me , I would just remove this next major comment. I will get to this directory with my next Pr on master, and I will see what is the correct typehint, and discuss it there

Reply:
Done!